package.json

```diff
@@ -137,11 +137,11 @@
 	"devDependencies": {
 		"@apollo/react-testing": "^3.1.3",
 		"@babel/plugin-proposal-class-properties": "^7.8.3",
-		"@babel/plugin-transform-runtime": "^7.8.3",
+		"@babel/plugin-transform-runtime": "7.9.0",
 		"@babel/preset-env": "^7.9.0",
-		"@babel/preset-react": "^7.8.3",
-		"@babel/preset-typescript": "^7.8.3",
-		"@babel/runtime-corejs3": "^7.8.4",
+		"@babel/preset-react": "7.9.1",
+		"@babel/preset-typescript": "7.9.0",
+		"@babel/runtime": "7.9.0",
```

Comment:
these updates were made based on this:
- https://github.com/facebook/create-react-app/issues/8680
and
- https://github.com/facebook/create-react-app/pull/8681/files

Reply:
also I noticed they use `@babel/runtime`, not `@babel/runtime-corejs3`, updated that as well, will double check on ejected CRA